site/docs/v1/tech/tutorials/integrate-dotnet.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: doc 
+title: Integrate Your C# .NET Application With FusionAuth 
+description: Integrate your C# .NET application with FusionAuth 
+navcategory: getting-started
+prerequisites: C# and dotnet
+technology: .NET 7
+language: C#
+---
+
+## Integrate Your {{page.technology}} Application With FusionAuth
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %} 
+
+Although this guide shows how to build the {{page.technology}} application using command line tools, you can also use [Visual Studio](https://visualstudio.microsoft.com) to build the project.
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but you’re going to use the {{page.language}} client library. The below instructions use `dotnet` from the command line, but you can use the client library with an IDE of your choice as well.
+
+First, create a {{page.technology}} project in a new directory:
+
+```shell
+dotnet new console --output SetupFusionauth && cd SetupFusionauth
+```
+
+If you want, you can [login to your instance](http://localhost:9011) and examine the new application configuration the script created for you.
+
+Now, copy and paste the following code into `Program.cs`.
+
+```csharp
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-example-client-libraries/main/dotnet/Program.cs %}
+```
+
+Then, you’ll need to import a few NuGet packages:
+
+```shell
+dotnet add package JSON.Net # for debugging
+dotnet add package FusionAuth.Client # for our client access
+```
+
+You can then publish and run the application. Replace `<YOUR_API_KEY>` with the API key that you generated.
+
+```shell
+dotnet publish -r osx-x64
+fusionauth_api_key=<YOUR_API_KEY> bin/Debug/net7.0/osx-x64/publish/SetupFusionauth 
+```
+
```

Comment:
Can you pull over the note about powershell here from the flask tutorial?

Reply:
Should add a note that the path to the SetupFusionauth executable will be different depending on your platform.